packages/clerk-expo-passkeys/README.md

```diff
@@ -0,0 +1,58 @@
+# Running the Example Project with Clerk on Your iOS Simulator or Device
```

Comment:
This is the README file that will be published on npm as the main documentation of this package. It's a public facing package, so we should add instructions on how to use this package. And then, we can also append this section on how to run the example project.

Reply:
Please check again, I updated the Readme.